### DISCUSSION TITLE
docs/faq

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#68] [Giscus]
# docs/faq

Frequently Asked Questions These frequently asked questions are collected from issues, emails and chats. Feel free to ask more questions via issue, email, Discord (for English) or QQ (for Chinese).

https://gorse.io/docs/faq.html

**Comment by albina111111** [comment]
Our UGS platform recommends short stories for users and users can create these stories and put tags to them. Please help me to choose optimal quality of tags for putting in UI to provide the best perfomance of the algorythm
